README.md

```diff
@@ -4,6 +4,19 @@ Module authors: [Komal S. Rathi](https://github.com/komalsrathi/),
                 [Zhuangzhuang Geng](https://github.com/zzgeng),
                 [Jo Lynne Rokita](https://github.com/jharenza)
 
+## Clone the repository
+
+To clone the repository, run:
+```
+git clone git@github.com:d3b-center/hope-cohort-analysis.git
+```
+
+## Download the data
+
+To download the current data release:
+```
+bash download_data.sh
```

Comment:
The file name is "download-data.sh" with a hyphen rather than underscore.

Reply:
good catch!